src/constants.py

```diff
@@ -3,39 +3,83 @@
     'citybucks': 'CB1 City Bucks',
     'cornell_card': 'CC1 Cornell Card',
     'laundry': 'LAU Sem Laundry'
-    # 'swipes': TBD
 }
+BRB_ONLY = 'brb_only'
 CORNELL_DINING_URL = 'https://now.dining.cornell.edu/api/1.0/dining/eateries.json'
 CORNELL_INSTITUTION_ID = '73116ae4-22ad-4c71-8ffd-11ba015407b1'
+DINING_HALL = 'dining_hall'
+EATERY_DATA_PATH = './eatery-data/'
 GET_URL = 'https://services.get.cbord.com/GETServices/services/json'
 GIT_CONTENT_URL = 'https://raw.githubusercontent.com/cuappdev'
 IGNORE_LOCATIONS = ['BS-No Bill Workstation', 'Admin Workstation (B)']
 IMAGES_URL = GIT_CONTENT_URL + '/assets/master/eatery/eatery-images/'
+TRILLIUM = 'trillium'
+ISOLATE_SWIPES = [
+    'date',
+    'session_type',
+    'weekday',
+    'location',
+    'start_time',
+    'end_time',
+    DINING_HALL,
+    BRB_ONLY,
+    TRILLIUM
+]
+ISOLATE_DATE = [
```

Comment:
i know you explained this, but the `ISOLATE` variables aren't intuitive to someone reading the code for the first time, the `ISOLATE` variables either need comments or better names. 

Also, why not define one list that has the union of these elements, and then build the three `ISOLATE` lists by copying the master list, and removing the unnecessary ones?

Reply:
ooh I dig that idea